packages/editor/package.json

```diff
@@ -46,10 +46,9 @@
 		"dom-react": "^2.2.1",
 		"dom-scroll-into-view": "^1.2.1",
 		"element-closest": "^2.0.2",
+		"http-build-query": "^0.7.0",
 		"lodash": "^4.17.10",
 		"memize": "^1.0.5",
-		"querystring": "^0.2.0",
```

Comment:
Aside: [`querystring` is a built-in](https://nodejs.org/api/querystring.html), so we never should have been defining it as a dependency.

Reply:
Well, it's not black or white here because depending on the bundler used by the packages consuming this package, we should define it or not.

Reply:
Hmm, an interesting point. I'm still not sure it's needed, since what's published to npm should be _targeting Node_ (CommonJS, built-ins), even if ultimately it's consumed in a web context by a bundler.